user-manager-service/src/main/java/com/crmfoodestablishment/user_auth_service/ApiErrorInfo.java

```diff
@@ -0,0 +1,20 @@
+package com.crmfoodestablishment.user_auth_service;
```

Comment:
I do believe that ApiErroInfo class should be moved to another one (where u use it)

Reply:
It is intended also to be used in handling of exceptions that occur in user manager part. So it serves as app-wide API error template